packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -95,22 +95,23 @@ contract PluginRepo is
         address _pluginFactory,
         bytes calldata _contentURI
     ) external auth(address(this), CREATE_VERSION_PERMISSION_ID) {
-        // Check if `_pluginFactory` is a `PluginFactoryBase` contract
+        // Check if `_pluginFactory` is a `PluginManager` contract
         if (!Address.isContract(_pluginFactory)) {
```

Comment:
Variables are still named `_pluginFactory` despite the renaming to `PluginManager`. 
Despite rushing, let's at least be consistent...